addons/mail/static/src/components/chatter/chatter.js

```diff
@@ -103,6 +107,17 @@ class Chatter extends Component {
         this.chatter.update({ isComposerVisible: false });
     }
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onScrollPanelScroll(ev) {
+        if (!this._threadRef || !this._threadRef.comp) {
```

Comment:
From how we use our other refs, apparently only `.comp` can be `undefined`, the ref object itself will always be set.